test/suite/test_live_restore01.py

```diff
@@ -48,7 +48,9 @@ def expect_success(self, config_str):
     def expect_failure(self, config_str, expected_error):
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
             lambda: self.open_conn("DEST", config=config_str), expected_error)
-        # No need to clean up the destination as WiredTiger failed to open.
```

Comment:
This comment wasn't true. The last test step `self.expect_failure("salvage=true ...` was leaving orphan `WiredTiger.lock` files in the DEST folder.